actionview/lib/action_view/lookup_context.rb

```diff
@@ -132,6 +132,11 @@ def exists?(name, prefixes = [], partial = false, keys = [], **options)
       end
       alias :template_exists? :exists?
 
+      def template_exists_in_any_format?(name, prefixes = [], partial = false, keys = [], **options)
+        options_with_all_formats = options.merge(formats: default_formats)
+        @view_paths.exists?(*args_for_lookup(name, prefixes, partial, keys, options_with_all_formats))
+      end
```

Comment:
I think I'm just passing in arguments required by ViewPaths. I figured that if the method might be reused elsewhere, might as well provide correct options. Don't think we'll be able to make this a private method, because we need to access this `ActionView` method from `ActionPack`.